src/pages/reference/combining-multiple-sources.md

```diff
@@ -48,7 +48,7 @@ query bestSellersByStore {
 
 The "Authors" Source is a gRPC API: [`authors-service`](https://github.com/charlypoly/graphql-mesh-docs-first-gateway/tree/master/packages/authors-service).
 
-We will use the `grpc` Handler with the `@graphql-mesh/grpc` package and configure it in our [`.meshrc.yaml`](https://github.com/charlypoly/graphql-mesh-docs-first-gateway/tree/master/packages/multiple-sources/.meshrc.yaml) file:
+We will use the `grpc` Handler with the `@graphql-mesh/grpc` package and configure it in our [`.meshrc.json`](https://github.com/charlypoly/graphql-mesh-docs-first-gateway/tree/master/packages/multiple-sources/.meshrc.yaml) file:
```

Comment:
Per my comment in #82, consider renaming all occurrences of `.meshrc.json` to `mesh.json`, to be consistent with your getting started guide:

* https://developer.adobe.com/graphql-mesh-gateway/gateway/create-mesh/#create-a-mesh-1